docker-compose-local-dev.yml

```diff
@@ -0,0 +1,13 @@
+# This version of the docker-compose file:
+#   - only runs the db in docker, allowing "easier" local development
+#   - can be run via `docker-compose -f docker-compose-local-dev.yml up`
```

Comment:
I am not able to get the app to build using the regular docker-compose file. There is a ruby version mismatch and the database doesn't connect for me either.

Where is that docker image maintained?

Reply:
`build: .` is telling Docker Compose to build a Docker image from https://github.com/citizenlabsgr/adopt-a-drain/blob/master/Dockerfile.

Reply:
I see, that should be updated to specify `2.5.3` instead of just `2.5`, otherwise the build will fail every time a new version of ruby comes out (I think, that's what happened to me).